frontend/src/services/applicationService.js

```diff
@@ -0,0 +1,22 @@
+import { ApiRoutes } from '@/utils/constants'
+import ApiService from '@/common/apiService'
+
+export default {
+  sortApplications(applications) {
+    applications?.sort((app1, app2) => {
+      const latestActivityDateOrder = new Date(app2.latestActivity) - new Date(app1.latestActivity)
+      return latestActivityDateOrder
```

Comment:
This is personal preference but I won't typically define a variable if it's only used once. At that point your code is one line and you can use an implicit return.

Reply:
it is updated